big_scape/enums/genbank.py

```diff
@@ -5,6 +5,6 @@
 
 class RECORD_TYPE(Enum):
     REGION = "region"
-    CANDIDATE_CLUSTER = "cand_cluster"
-    PROTO_CLUSTER = "proto_cluster"
+    CAND_CLUSTER = "cand_cluster"
+    PROTOCLUSTER = "protocluster"
```

Comment:
I see what you're going for, but let's keep the variable naming according to the PEP8 style guide; for enums this is capitals and underscores.

CAND_CLUSTER is fine